Dockerfile

```diff
@@ -8,8 +8,9 @@ COPY ./requirements.txt /app/
 
 RUN apt-get update && apt-get install -y git
 RUN pip install --upgrade pip && pip install -r /app/requirements.txt
-RUN pip install -f https://download.pytorch.org/whl/torch_stable.html torch==1.10.0+cpu
-RUN pip install spotlight@git+https://github.com/maciejkula/spotlight.git@master
+RUN pip install -f https://download.pytorch.org/whl/torch_stable.html torch==1.12.1+cpu
```

Comment:
🤔 Have you made sure that this version change shouldn't break anything? Like any pytorch version release notes or anything like that?

Reply:
Yep, the CF job/Spotlight still runs fine with the new version of PyTorch!